docs/source/test_set.rst

```diff
@@ -34,4 +35,22 @@ To run the test set on a pre-trained model, use this method.
     trainer.test(model)
 
 In this  case, the options you pass to trainer will be used when
-running the test set (ie: 16-bit, dp, ddp, etc...)
\ No newline at end of file
+running the test set (ie: 16-bit, dp, ddp, etc...)
+
+
+Test additional data loaders
+----------------------------
```

Comment:
```suggestion
Test with additional data loaders
---------------------------------
```
(since we are not testing the dataloaders themselves, but the data they provide)